docs/content/get-started.md

```diff
@@ -57,6 +57,20 @@ Set permissions on the OPA executable:
 chmod 755 ./opa
 ```
 
+(Optional) Add the OPA executable to your PATH for global access:
+
+Add the following to your .bash_profile
+
+```shell
+export PATH=/path/containing/executable:$PATH
```

Comment:
We should use `export PATH=$PATH:/path/containing/executable` to avoid accidentally changing other behavior on the system (the paths are searched in order, by putting the new part at the end it looks there last).

Reply:
Thanks for correcting that.